### PR TITLE
[Common] change caps format for other/tensor

### PR DESCRIPTION
**Pull request by jaeyun-jung**
This PR changes caps format for other/tensor.
Caps for tensor stream : other/tensor, framerate = (fraction) [0, max], type = (string) { uint8, .. }, dimension = (string) d1:d2:d3:d4

Related issue : #1027

Signed-off-by: Jaeyun Jung <jy1210.jung@samsung.com>

### CODE REVIEW

**Comment by jaeyun-jung** [comment]
We have to fix testcases after merging PR #1030 and #1031

**Comment by myungjoo** [comment]
This changes the core protocol. PTAL: @leemgs @jijoongmoon @again4you @wooksong @helloahn 
This is going to be the final major changes for 0.1.0 release.

File: gst/nnstreamer/tensor_common.h [hunk 1]
```diff
@@ -85,20 +85,18 @@ G_BEGIN_DECLS
...
-    "dim1 = " GST_TENSOR_DIM_RANGE ", " \
-    "dim2 = " GST_TENSOR_DIM_RANGE ", " \
-    "dim3 = " GST_TENSOR_DIM_RANGE ", " \
-    "dim4 = " GST_TENSOR_DIM_RANGE ", " \
-    "type = (string) " GST_TENSOR_TYPE_ALL ", " \
     "framerate = " GST_TENSOR_RATE_RANGE
-
+    /**
+     * type should be one of types in GST_TENSOR_TYPE_ALL
+     * "type = (string) uint8"
```
Comment (jijoongmoon) [inline gst/nnstreamer/tensor_common.h]: Is there any reason to set type as hidden?

File: gst/nnstreamer/tensor_common.h [hunk 2]
```diff
@@ -85,20 +85,18 @@ G_BEGIN_DECLS
     GST_TENSOR_TEXT_CAPS_STR "; " \
     GST_TENSOR_OCTET_CAPS_STR
 
-/** @todo I'm not sure if the range is to be 1, 65535 or larger */
-#define GST_TENSOR_DIM_RANGE "(int) [ 1, 65535 ]"
-#define GST_TENSOR_RATE_RANGE "(fraction) [ 0/1, 2147483647/1 ]"
+#define GST_TENSOR_RATE_RANGE "(fraction) [ 0, max ]"
```
Comment (jijoongmoon) [inline gst/nnstreamer/tensor_common.h]: What is the difference with "#define GST_TENSOR_RATE_RANGE "(fraction) [ 0/1, 2147483647/1 ]""

Comment (jaeyun-jung) [reply gst/nnstreamer/tensor_common.h]: the value is not different, indeed. (0 ~ 2147483647)

File: gst/nnstreamer/tensor_common.h (hunk 1 above)
Comment (jaeyun-jung) [reply gst/nnstreamer/tensor_common.h]: Right, for the consistency with other/tensors.

**Pull request merged**